CHANGELOG.md

```diff
@@ -14,6 +14,8 @@
     * Added little helper tool to compute mt to nuclear ratios for NGS data.
 * [**mosdepth**](https://github.com/brentp/mosdepth)
     * fast BAM/CRAM depth calculation for WGS, exome, or targeted sequencing
+* [**MultiVCFAnalyzer**](https://github.com/alexherbig/multivcfanalyzer)
+    * combining multiple VCF files into one coherent report and format for downstream analysis.
```

Comment:
I think a better description would be: `SNP filtering and SNP alignment generation of GATK UnifiedGenotyper VCF files.`